test/frames.ts

```diff
@@ -162,4 +163,123 @@ describe('NodeID3 frames', function () {
         const readTags = NodeID3.read(createdBuffer, { noRaw: true})
         assert.deepStrictEqual(tags, readTags)
     })
+
+    describe('read() does not match create()', function() {
+        it('create throws', function() {
+            const throwingTags = [
+                { POPM: {} },
+                { POPM: {
+                    email: 'test'
+                }},
+                { POPM: {
+                    email: 'test',
+                    rating: 1
+                }},
+                { CTOC: {} },
+                { USLT: {} },
+                { CHAP: {} },
+                { COMM: {} },
+                { TALB: null },
+                { WCOM: null },
+                { APIC: {
+                    mime: "a",
+                    type: {
+                        id: TagConstants.AttachedPicture.PictureType.FRONT_COVER
+                    },
+                    description: "d",
+                    imageBuffer: ""
+                }},
+                { COMM: {
+                    language: 'asdf',
+                    text: 'text'
+                }},
+                { COMR: {
+                    prices: {
+                        EURO: 13
+                    }
+                }}
+            ]
+
+            for(const throwingTag of throwingTags) {
+                expect(() => NodeID3.create(throwingTag as never)).to.throw()
+            }
+        })
+
+        it('frame builder changes data', function() {
+            const tags = {
+                unsynchronisedLyrics: 'just a string',
+                commercialFrame: {
+                    validUntil: { year: 2023, month: 9, day: 'a'},
```

Comment:
`day` is not valid shouldn't it throw?

Reply:
I think it would be more in line with the other throws? Right now, it's just ignored

Reply:
Ah yes, I remember, there is no validation right now, we should probably add one after this PR.